packages/design-system/src/components/Table/TableDataCell.jsx

```diff
@@ -11,19 +10,8 @@ export const TableDataCell = ({
   stackedTitle,
   ...others
 }) => {
-  const tableStackableContext = useContext(TableContext);
-  if (process.env.NODE_ENV !== 'production') {
```

Comment:
Hm its a bit sad that we lose these warnings. Is it possible to use `React.cloneElement` in the root table component in order to pass an prop to the children that replaces `tableStackableContext`?

Reply:
Actually, I'm ok with this being handled in the other table follow up PR, as we need this PR to be merged soon, before the next release